src/main/java/core/basesyntax/RandomColor.java

```diff
@@ -1,10 +1,22 @@
 package core.basesyntax;
 
+import core.basesyntax.interfaces.SettingRandomColor;
+
+import java.util.Random;
+
 /**
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод для случайного
  * выбора цвета</p>
  */
 public class RandomColor {
     private static final String[] COLORS = {"red", "green", "blue", "yellow", "pink",
             "black", "white"};
+
+    public String get() {
+        SettingRandomColor setColor = () -> {
+            int random = new Random().nextInt(COLORS.length);
```

Comment:
try to solve without creating a `random` variable

Reply:
done